common/main/java/com/couchbase/lite/AbstractReplicatorConfiguration.java

```diff
@@ -21,6 +21,9 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import okhttp3.internal.Util;
```

Comment:
It's internal!. I'm fine with this but just want to point it out. We probably needs to change a bunch of things when upgrading okhttp anyway incase the internal classes are gone.

Reply:
Yes... It's a very, very fair point.

It was my opinion that it was less fragile to use the *exact* check that they use, than to try to duplicate a possibly moving target.  Absolutely open to opinions to the contrary